src/logo.tsx

```diff
@@ -0,0 +1,359 @@
+            <path d="M40.2002 63.3896C40.2002 64.916 39.1202 65.9996 37.6394 65.9996C36.1586 65.9996 35.0846 64.9196 35.0846 63.3896C35.0846 61.8596 36.1646 60.7808 37.6442 60.7808C39.1238 60.7808 40.2002 61.8644 40.2002 63.3896ZM38.969 63.3896C38.969 62.504 38.3786 61.9136 37.6394 61.9136C36.9002 61.9136 36.3098 62.504 36.3098 63.3896C36.3098 64.2752 36.9014 64.8668 37.6394 64.8668C38.3774 64.8668 38.969 64.2764 38.969 63.3896Z" />
+            <path d="M40.9382 60.8792H42.0698L42.1682 61.3712H42.2174C42.3154 61.2584 42.428 61.1591 42.5522 61.076C42.8637 60.8774 43.2268 60.7747 43.5962 60.7808C44.729 60.7808 45.5654 61.6208 45.5654 62.8976V65.8976H44.3354V62.9996C44.3354 62.7122 44.2213 62.4366 44.018 62.2334C43.8148 62.0302 43.5392 61.916 43.2518 61.916C42.9644 61.916 42.6888 62.0302 42.4856 62.2334C42.2824 62.4366 42.1682 62.7122 42.1682 62.9996V65.9048H40.9382V60.8792Z" />
+            <path d="M53.4001 64.0304C53.2429 64.8176 52.6225 65.9996 51.0373 65.9996C49.5602 65.9996 48.4766 64.9196 48.4766 63.3896C48.4766 61.8596 49.5566 60.7808 51.0373 60.7808C52.6225 60.7808 53.2429 61.9628 53.4001 62.7008H52.1689C52.0913 62.4646 51.9389 62.2601 51.7347 62.1181C51.5306 61.9761 51.2858 61.9043 51.0373 61.9136C50.2982 61.9136 49.7078 62.504 49.7078 63.3896C49.7078 64.2752 50.2982 64.8668 51.0373 64.8668C51.2939 64.8785 51.5466 64.8013 51.7527 64.6483C51.9589 64.4953 52.1059 64.2757 52.1689 64.0268L53.4001 64.0304Z" />
+            <path d="M57.1225 65.4092H57.0733C56.974 65.5247 56.8574 65.6243 56.7277 65.7044C56.4037 65.9048 56.0287 66.0073 55.6477 65.9996C54.5053 65.9996 53.8262 65.3 53.8262 64.4732C53.8262 63.4892 54.515 62.7992 55.8937 62.7992H57.0265V62.7008C57.0265 62.1488 56.6917 61.7648 56.1397 61.7648C55.5877 61.7648 55.2997 62.1248 55.2541 62.4056H54.0206C54.1286 61.5788 54.7981 60.7808 56.1373 60.7808C57.4177 60.7808 58.2541 61.628 58.2541 62.7008V65.9012H57.2209L57.1225 65.4092ZM57.0241 63.7292H55.9897C55.3501 63.7292 55.0538 63.9692 55.0538 64.3688C55.0538 64.7684 55.3393 65.0084 55.8421 65.0084C56.5897 65.0084 57.0241 64.5752 57.0241 63.8768V63.7292Z" />
+            <path d="M46.5086 60.8792H47.7386V65.9012H46.5086V60.8792Z" />
+            <path d="M47.1242 60.1569C47.5265 60.1569 47.8526 59.8308 47.8526 59.4285C47.8526 59.0262 47.5265 58.7001 47.1242 58.7001C46.7219 58.7001 46.3958 59.0262 46.3958 59.4285C46.3958 59.8308 46.7219 60.1569 47.1242 60.1569Z" />
+            <path d="M38.0834 58.7624H39.3626L38.1806 60.1904H37.1475L38.0834 58.7624Z" />
+        </svg>
+    );
+};
+
+const BlauLogoImage = ({size, logoType}: LogoImageProps) => {
+    const {isDarkMode} = useTheme();
+    const isInverse = useIsInverseVariant();
+    const color = isInverse && !isDarkMode ? vars.colors.inverse : vars.colors.brand;
+
+    if (logoType === 'vertical') {
+        return (
+            <svg height={size} viewBox="0 0 73 72" fill={color}>
+                <path d="M56.42 52.2003H18.2V63.0003H56.42V52.2003Z" fill="#7814B2" />
```

Comment:
To keep Blau icon with original colors  (the logo include two colors in default version)
![image](https://user-images.githubusercontent.com/6722153/232874427-818c9f88-3d75-4ea8-b947-b8f4fc6563ff.png) ![image](https://user-images.githubusercontent.com/6722153/232874445-40251cd3-e84e-47d0-a561-bd1fa3b77aad.png)


my proposal
`const colorSecondary = isInverse && !isDarkMode ? vars.colors.inverse : vars.colors.promo;`

`<path d="M56.42 52.2003H18.2V63.0003H56.42V52.2003Z" fill={colorSecondary} />`